demisto_sdk/commands/common/hook_validations/incident_field.py

```diff
@@ -301,3 +303,12 @@ def is_cliname_is_builtin_key(self):
                 f"{self.file_path}: cliName field can not be {cliname} as it's a builtin key."
             )
         return is_valid
+
+    def is_valid_from_version(self):
+        server_base_version = str(self.current_file.get('fromVersion', '0.0.0')).split('.')[0]
```

Comment:
you should catch `ValueError` exception if in case the it will to translate it to integer, and return an error that say the version is invalid and print it.

And add a test for that please.